src/redux/data.ts

```diff
@@ -549,7 +1085,9 @@ export const addressAssetsReceived = (
     });
   }
 
-  let parsedAssets = parseAccountAssets(updatedAssets, uniqueTokens);
+  let parsedAssets = parseAccountAssets(updatedAssets, uniqueTokens) as {
+    [id: string]: ParsedAddressAsset;
+  };
```

Comment:
Until `src/parsers` is typed, casting like this is helpful, since otherwise the return type of `parseAccountAssets` doesn't seem to account for the spread operation and is missing some fields.